cartofacade/queue.go

```diff
@@ -0,0 +1,194 @@
+// Package cartofacade contains the api to call into CGO
```

Comment:
Could we please also comment that this package exists to ensure that only one go routine is calling into the CGO api at a time to ensure the go runtime doesn't spawn multiple OS threads, which would harm performance?